statusHandler/prometheusHandler.go

```diff
@@ -0,0 +1,105 @@
+package statusHandler
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/prometheus/client_golang/prometheus"
+	prometheusUtils "github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+// PrometheusStatusHandler will define the handler which will update prometheus metrics
+type PrometheusStatusHandler struct {
+}
+
+var prometheusGaugeMetrics map[string]prometheus.Gauge
+
+// InitMetrics will declare and init all the metrics which should be used for Prometheus
+func (psh *PrometheusStatusHandler) InitMetrics() {
+	prometheusGaugeMetrics = make(map[string]prometheus.Gauge)
+
+	erdSynchronizedRound := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: core.MetricSynchronizedRound,
+		Help: "todo",
```

Comment:
should remove "todo" with something useful

Reply:
sorry, forgotted them. done